pkg/controller/mizar/mizar-node-controller.go

```diff
@@ -0,0 +1,293 @@
+/*
+Copyright 2020 Authors of Arktos.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Reference:
+(1) https://github.com/h-w-chen/arktos.git, arktos/cmd/arktos-network-controller
```

Comment:
Don't put https://github.com/h-w-chen/arktos.git in the code, it is Hongwei's private repo

Reply:
> Don't put https://github.com/h-w-chen/arktos.git in the code, it is Hongwei's private repo

This repository is deleted.